CHANGELOG.md

```diff
@@ -37,10 +37,10 @@
 * [#11459](https://github.com/rubocop/rubocop/pull/11459): Make `Lint/UselessRuby2Keywords` aware of conditions. ([@splattael][])
 * [#11415](https://github.com/rubocop/rubocop/issues/11415): Fix a false positive for `Lint/UselessMethodDefinition` when method definition contains rest arguments. ([@koic][])
 * [#11418](https://github.com/rubocop/rubocop/issues/11418): Fix a false positive for `Style/MethodCallWithArgsParentheses` when using anonymous rest arguments or anonymous keyword rest arguments. ([@koic][])
-* [#11431](https://github.com/rubocop/rubocop/pull/11431): Fix a crash in Style/HashSyntax. ([@gsamokovarov][])
+* [#11431](https://github.com/rubocop/rubocop/pull/11431): Fix a crash in `Style/HashSyntax`. ([@gsamokovarov][])
```

Comment:
Errors were detected by tweaking `Chanelog` examples in `project_spec.rb` with:

```ruby
  describe 'Changelog' do
    subject(:changelog) { File.read(path) }

    let(:obsolete_cop_names) do
      yaml = YAML.load_file("config/obsoletion.yml")

      yaml.fetch("renamed").keys + 
        yaml.fetch("split").keys + 
        yaml.fetch("removed").keys
    end

    let(:path) { File.expand_path('../CHANGELOG.md', __dir__) }
    let(:entries) { lines.grep(/^\*/).map(&:chomp) }

    include_examples 'has Changelog format'

    it 'has valid cop name with backticks', :aggregate_failures do
      entries.each do |entry|
        entry.scan(%r{\b[A-Z]\w+(?:/[A-Z]\w+)+\b}) do |cop_name|
          expect(all_cop_names.include?(cop_name)).to be(true), "Invalid cop name #{cop_name}. #{entry}"
          expect(entry.include?("`#{cop_name}`")).to be(true), "Missing backticks for #{cop_name}. #{entry}"
        end
      end
    end
```

I think it's unnecessary to check `CHANGELOG.md` permanently because we are now checking `changelog/` automatically.